src/Traits/Thumb.php

```diff
@@ -26,7 +26,7 @@ public function Thumb($width, $height, $gravity = null)
             $gravity = $customGravity;
         }
 
-        $clone->asset->resize(Resize::thumbnail($width, $height, $gravity));
+        $clone->asset?->resize(Resize::thumbnail($width, $height, $gravity));
```

Comment:
what is the `?`?

Reply:
Null safe operator. basically if asset is null it won't try to resize it.